python/lsst/ts/m2gui/main_window.py

```diff
@@ -200,26 +206,38 @@ def _get_log_file_name(self):
             pathlib.Path(__file__).parents[0] / ".." / ".." / ".." / ".." / "log" / name
         )
 
-    def _set_model(self, is_simulation_mode):
+    def _set_model(self, is_simulation_mode, is_crio_simulator):
```

Comment:
This reminded me of [this article](https://betterprogramming.pub/dont-use-boolean-arguments-use-enums-c7cd7ab1876a) I saw some time ago. Basically he advocates that it is better to use enums instead of boolean parameters. It is pretty interesting. 

I don't think you need to implement this now, but consider this for future updates. Basically, replace the boolean arguments with enumeration.

Reply:
Let me think about this. Another possibility might be to use the `mypy` annotation to tell the user this variable is a boolean. Thanks for the suggestion!